queue_services/entity-filer/src/entity_filer/filing_processors/correction.py

```diff
@@ -51,9 +50,7 @@ def process(correction_filing: Filing, filing: Dict, filing_meta: FilingMeta, bu
     )
 
     corrected_filing_type = filing['correction']['correctedFilingType']
-    is_sr_correction = business.legal_type == 'CP' and is_special_resolution_correction_by_filing_json(filing)
-    if (business.legal_type in ['SP', 'GP', 'BC', 'BEN', 'CC', 'ULC'] or
-            is_sr_correction) and \
```

Comment:
Previously we needed the `is_sr_correction` check in particular to avoid local CP filings like change of address(COA) & change of directors(COD) from being processed as we didn't support that yet.

I imagine this is still an issue?  Or have we made updates to support change of address and change of directors for CP corrections?

Reply:
I think there might be some requirements around change of directors and it's done in the filing now for  SR/CP? Jia would have to comment

Reply:
yes you correct, I have some misunderstanding of correction of address on CP and COA. I think we need leave is_sr_correction here for now.

Reply:
@seeker25 yeah, I'm talking about a correction on a COA or COD as a staff user.  It's done as a manual process(local correction) currently.  It puts these types of corrections into a filing state of PENDING_CORRECTION which needs to be taken into account in this code.

Here's an example of a COD correction I just did now in DEV.

![image](https://github.com/bcgov/lear/assets/6685223/007ad053-c625-443e-b06c-5569827a13ab)

![image](https://github.com/bcgov/lear/assets/6685223/dbd63a64-5d99-4a5b-8d16-579c85a61a98)

![image](https://github.com/bcgov/lear/assets/6685223/57f4c106-b88e-4bfa-ad90-0f15f9db25b7)

Reply:
@seeker25 @argush3 
Thanks for bringing it out, I just meet with Mihai, Ethan and Dana about this question. So the COA and COD is not an issue, we can change it to generic correction now.

Reply:
Awesome!

Reply:
So just to be clear, what is the plan for this piece of code right now?  The UI still allows COA and COD corrections right now.  Are we going to remove COA & COD correction options and change over to the generic correction before removing the `is_sr_correction` check?

Reply:
I think Jia is on vacation now, I'll have to take over and look into this deeper tomorrow.

Reply:
As discussed with Mihai and Argus - we're going to leave this ticket open until we can test it with directors.